src/main/java/org/zalando/nakadi/service/subscription/zk/ZkSubscriptionClient.java

```diff
@@ -59,14 +61,13 @@
      * |  |- 1: {session: null, next_session: null, state: UNASSIGNED}
      * |   |- offset: {OFFSET}
      *
-     * @param partitionOffsets Data to use for subscription filling.
+     * @param cursors Data to use for subscription filling.
      */
-    void fillEmptySubscription(Map<Partition.PartitionKey, String> partitionOffsets);
+    void fillEmptySubscription(Collection<NakadiCursor> cursors);
```

Comment:
No, and it was not for a while. I do not want to touch it here, cause I will refactor it in upcoming task